### PR TITLE
Add logos for Abachi tokens - ABI, sABI and gABI

### PR DESCRIPTION
**Pull request by 0xigor**
## Context (issues, jira)

Adding logos for Abachi tokens to ledger

## Description / Usage

Add logos for Abachi tokens - sABI, ABI and gABI to ledger. Support for tokens was added to ledger live desktop already

Contract addresses (polygon)-
ABI   - `0x6d5f5317308C6fE7D6CE16930353a8Dfd92Ba4D7`
sABI - `0x925a785a347f4a03529b06C50fa1b9a10808CAb5`
gABI - `0xEd6AAb1615AaC7BC4C108dFd4Fdc9AD0c8304d47`

## Expectations

- [ ] **Test coverage: The changes of this PR are covered by test.** Unit test were added with mocks when depending on a backend/device.
- [x] **No impact: The changes of this PR have ZERO impact on the userland.** Meaning, we can use these changes without modifying LLD/LLM at all. It will be a "noop" and the maintainers will be able to bump it without changing anything.

### CODE REVIEW

File: src/data/icons/svg/ABI.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<svg fill="none" viewBox="0 0 24 24">
+  <path fill="url(#a)" fill-rule="evenodd" d="M12 24c6.627 0 12-5.372 12-12 0-6.627-5.373-12-12-12-4.033 0-7.601 1.99-9.777 5.04h4.44a.96.96 0 0 1 0 1.92H1.106C.891 7.425.704 7.905.55 8.4h13.173a.96.96 0 1 1 0 1.92H.116A12.098 12.098 0 0 0 0 11.915h4.687a.96.96 0 1 1 0 1.92H.139c.08.52.193 1.03.337 1.525h13.246a.96.96 0 1 1 0 1.92H1.221c.283.578.612 1.13.982 1.651h4.46a.96.96 0 1 1 0 1.92H3.897A11.957 11.957 0 0 0 12 24.002zm4.358-6.72a.96.96 0 1 1 0-1.92.96.96 0 0 1 0 1.92zm-.96-7.92a.96.96 0 1 0 1.92 0 .96.96 0 0 0-1.92 0zm-7.21 10.538a.96.96 0 1 0 1.92 0 .96.96 0 0 0-1.92 0zm.96-12.938a.96.96 0 1 1 0-1.92.96.96 0 0 1 0 1.92zm.042 5.915c0 .53.43.96.96.96h10.797a.96.96 0 1 0 0-1.92H10.15a.96.96 0 0 0-.96.96zm-1.778.96a.96.96 0 1 1 0-1.92.96.96 0 0 1 0 1.92z" clip-rule="evenodd"/>
+  <defs>
+    <linearGradient id="a" x1="19.68" x2="11.847" y1="3" y2="23.943" gradientUnits="userSpaceOnUse">
```
Comment (LFBarreto) [inline src/data/icons/svg/ABI.svg:4]: linear gradients, defs, masks, clip-path, filters and inline styles arent valid for our svg icons as we parse them to create react and react native components.
try comparing it to other icons in the repository to check for correct format.
only use svg, path, circle, ellipse, rect, polyline, line tags if needed and add fill="black" as props to them as we will inject color stylings during parsing

thx :pray:

**Review by LFBarreto — CHANGES_REQUESTED** [review]

**Comment by 0xigor** [comment]
@LFBarreto , The SVGs have been updated. Please check and let me know if any other changes are needed.

Thanks!

**Review by LFBarreto — APPROVED** [review]

**Comment by adrienlacombe** [comment]
@LFBarreto can we move this to the monorepo by any chance? Thank you!

**Comment by LFBarreto** [comment]
has been migrated towards https://github.com/LedgerHQ/ledger-live/pull/107
sorry for the delay on this :bow:

**Pull request closed without merging**